src/main/java/teammates/common/util/Const.java

```diff
@@ -259,7 +263,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_COURSE_PROFILE = "Your profile in this course";
         public static final String STUDENT_COURSE_DETAILS = "View and edit information regarding your team";
     
-        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING = "The session is not open for submission at this time. It is expected to open later.";
+        public static final String STUDENT_FEEDBACK_SESSION_STATUS_AWAITING =
+                "The session is not open for submission at this time. It is expected to open later.";
```

Comment:
Same as above:
Since it is just a string constant, should we start it in the same line as the `=` and break whenever we reach the limit?
If we do this consistently for all string constants, they'll just read like a justified text, it will and also help save brain power on deciding where to put the breaks.